### PR TITLE
Add support for conv3d

### PR DESCRIPTION
**Pull request by kaczmarj**
Hello,

This PR adds support for `conv3d`, which was implemented by https://github.com/tensorflow/tfjs-core/pull/1238.

Thank you,
Jakub

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-converter/260)

### CODE REVIEW

**Comment by kaczmarj** [comment]
I signed it!

File: src/operations/op_list/convolution.ts [hunk 1]
```diff
@@ -177,5 +177,36 @@ export const json = [
...
+      {'tfInputIndex': 0, 'dlParamName': 'x', 'type': 'tensor'},
+      {'tfInputIndex': 1, 'dlParamName': 'filter', 'type': 'tensor'}, {
+        'tfParamName': 'T',
+        'dlParamName': 'dtype',
+        'type': 'dtype',
+        'notSupported': true
+      },
+      {'tfParamName': 'strides', 'dlParamName': 'strides', 'type': 'number[]'},
+      {'tfParamName': 'padding', 'dlParamName': 'pad', 'type': 'string'}, {
+        'tfParamName': 'useCudnnOnGpu',
```
Comment (pyu10055) [inline src/operations/op_list/convolution.ts]: I don't think this op has `useCudnnOnGpu` attribute field.

File: src/operations/executors/convolution_executor.ts [hunk 2]
```diff
@@ -90,6 +90,24 @@ export let executeOp: OpExecutor =
...
+              (getParamValue('dataFormat', node, tensorMap, context) as string)
+                  .toUpperCase();
+          const dilations =
+              getParamValue('dilations', node, tensorMap, context) as number[];
+          return [tfc.conv3d(
+              getParamValue('x', node, tensorMap, context) as tfc.Tensor4D |
+                  tfc.Tensor<tfc.Rank.R5>,  // TODO: change type to tfc.Tensor5D
+              getParamValue('filter', node, tensorMap, context) as
+                  tfc.Tensor<tfc.Rank.R5>,  // TODO: change type to tfc.Tensor5D
+              [stride[1], stride[2], stride[3]], pad as 'valid' | 'same',
```
Comment (pyu10055) [inline src/operations/executors/convolution_executor.ts]: TF op has strides and dilation array in length of 5, tfjs-core only support length of 3, seems we only support dimension of [depth, height, width].
Should both of strides and dilation values selected from indices of 1, 2, 3?

Comment (kaczmarj) [reply src/operations/executors/convolution_executor.ts]: yes, i have changed the strides and dilations to both use indices 1, 2, 3. fyi according to the [`tf.nn.conv3d` docs](https://www.tensorflow.org/api_docs/python/tf/nn/conv3d), indices 0 and 4 in strides and dilations must be 1.

File: src/operations/op_list/convolution.ts (hunk 1 above)
Comment (kaczmarj) [reply src/operations/op_list/convolution.ts]: i removed this field

**Comment by pyu10055** [comment]
@kaczmarj Thanks for the contribution, please revisit this PR, since the Conv3D op has been added to core, but you need to run

```
yarn gen-json
```

to make the test pass.

**Pull request merged**